docs/packaging.md

```diff
@@ -0,0 +1,53 @@
+# Application Lifecycle
```

Comment:
Good question, Seems there is a difference between opensource version vs. paypal rocksqubs version ...